docs/IModelsClientAuthoring.md

```diff
@@ -0,0 +1,78 @@
+# `@itwin/imodels-client-authoring`
+
+`IModelsClient` exposed by `@itwin/imodels-client-authoring` package extends `IModelsClient` exposed by `@itwin/imodels-client-management` package so this documentation references sections from [`@itwin/imodels-client-management` documentation](IModelsClientManagement.md). All types mentioned there are also exported from `@itwin/imodels-client-authoring` package.
```

Comment:
`@itwin/imodels-client-authoring` package extends `IModelsClient` exposed by `@itwin/imodels-client-management` package thus this documentation references sections from [`@itwin/imodels-client-management` documentation](IModelsClientManagement.md)